src/core/electrostatics_magnetostatics/scafacos.cpp

```diff
@@ -269,26 +268,26 @@ void tune_r_cut() {
   auto const min_box_l = *boost::min_element(box_geo.length());
   auto const min_local_box_l = *boost::min_element(local_geo.length());
 
-  /** scafacos p3m and Ewald do not accept r_cut 0 for no good reason */
+  /* scafacos p3m and Ewald do not accept r_cut 0 for no good reason */
   double r_min = 1.0;
   double r_max = std::min(min_local_box_l, min_box_l / 2.0) - skin;
   double t_min = 0;
   double t_max = std::numeric_limits<double>::max();
 
-  /** Run bisection */
+  /* Run bisection */
   while (std::fabs(r_min - r_max) > tune_limit) {
     const double dr = 0.5 * (r_max - r_min);
     const double t_mid = time_r_cut(r_min + dr);
     t_min = time_r_cut(r_min);
     t_max = time_r_cut(r_max);
 
     if (t_min <= 0.0) {
-      r_min += tune_limit;
+      r_min += tune_limit; // NOLINT(clang-analyzer-deadcode.DeadStores)
```

Comment:
I think this _is_ a dead store, so it can be removed

Reply:
Obviously, but could it be a bug, e.g. someone accidentally adding a `break;`? I looked into the history, but it has been like this for several years.

Reply:
No I think it's fine, the `r_{min, max}` are only used internally for the bisection, which is terminated by the `break`. Adding the linter exception makes it look as if this was there on purpose, which will make somebody in the future spend time figuring out why this jean-noel dude in 2020 added an linter exception to keep a dead store :-)